### PR TITLE
Add new package: py-clamd

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-clamd/package.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyClamd(PythonPackage):
+    """clamd is a portable Python module to use the ClamAV anti-virus
+    engine on Windows, Linux, MacOSX and other platforms. It requires
+    a running instance of the clamd daemon."""
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-clamd/package.py:12]: Do we need to add a run-dep on `clamd`?

Comment (darmac) [reply var/spack/repos/builtin/packages/py-clamd/package.py:12]: Yes, but I haven't started the `clamd` patch yet.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-clamd/package.py:12]: Should we wait on this PR until there is a `clamd` package?

Comment (darmac) [reply var/spack/repos/builtin/packages/py-clamd/package.py:12]: Yes, I think so.

**Pull request closed without merging**